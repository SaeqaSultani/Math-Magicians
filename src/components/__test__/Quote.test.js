import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

it('Check any changes to the component', () => {
  const tree = renderer.create(
    <Quote />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('A div with className container should be rendered', () => {
  render(
    <Quote />,
  );
  const div = screen.getByTestId('quote');
  expect(div).toHaveClass('container');
});