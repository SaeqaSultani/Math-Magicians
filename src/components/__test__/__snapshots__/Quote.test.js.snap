// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check any changes to the component 1`] = `
<div
  className="container"
  data-testid="quote"
>
  <div
    className="wrapper"
  >
    Mathematics is not about numbers, equations, computations,
    <br />
    or algorithms: it is about understanding.â€“William Paul Thurston
  </div>
</div>
`;
